tests/rpc/rpc_01-system_test.go

```diff
@@ -77,7 +80,7 @@ func TestSystemRPC(t *testing.T) { //nolint:tparallel
 				return false, err // error and stop retrying
 			}
 
-			ok = healthResponse.Peers == numberOfNodes-1 && !healthResponse.IsSyncing
+			ok = healthResponse.Peers == numberOfNodes-1
```

Comment:
Are you sure about this change? I'm looking why into why it's flaky,

Reply:
I am investigating why the `healthResponse.IsSyncing` is always true when we use `westend-dev` but when we use `gssmr` the `healthResponse.IsSyncing` is false

Reply:
I fixed this by using `westend-local` instead of `westend-dev`